src/helpers/wallet/trezorTransformTransaction.ts

```diff
@@ -0,0 +1,194 @@
+// https://github.com/trezor/connect/blob/develop/src/js/plugins/stellar/plugin.js
```

Comment:
This file is taken from Trezor Connect. It looks like they might be using an older version of `stellar-sdk`. The Account Viewer they have is on v0.6.2 (about 4 years old).
I tried to add as many decent types as possible.